rtengine/iimage.h

```diff
@@ -98,6 +98,10 @@ class ImageDatas : virtual public ImageDimensions
         return "unknown";
     }
 
+    virtual bool isBW () const
```

Comment:
I guess for the same reason we use the anonymous namespace, to make the problem we pose to the linker as simple as possible by presenting the most restricted set of symbols we can work with so that it can concentrate on producing the most efficient binary.
But for RT in particular, yeah definitely also build time. :-( A full debug build already needs north of six minutes on my machine, so I a crave for every compile and link time reduction. And this only gets worse with LTO enabled which however is sadly missed for an "organically grown" code base like RT's.